packages/grid/src/vaadin-grid-mixin.js

```diff
@@ -1039,6 +1045,43 @@ export const GridMixin = (superClass) =>
       }
     }
 
+    /** @private */
+    __isTargetInViewport(target) {
```

Comment:
Could rename this to `__isCellFullyVisible` to make its purpose more obvious

Reply:
Done.